tvshows/src/components/features/review/StarRating.tsx

```diff
@@ -0,0 +1,66 @@
+import { Flex, Icon} from "@chakra-ui/react";
+import { StarIcon } from "@chakra-ui/icons";
+import { useState, useEffect } from "react";
+
+export interface IStarRating {
+  defaultValue: number;
+  onChange: (value: number) => void;
+  mode: number;
```

Comment:
`mode` could be more interactive. For example: `mode: 'interactive' | 'static'` or `isInteractive: boolean`